cookiecutter/cli.py

```diff
@@ -106,6 +134,9 @@ def main(
     if template == u'help':
         click.echo(click.get_current_context().get_help())
         sys.exit(0)
+    if template == u'installed' or template == u'i':
```

Comment:
Sounds great to me. I'm open to basically any name for the command. But it's a real pain to hunt down the template names over and over. Once you have them downloaded it would be nice to use it as a list of what you've used before.